sdk/modelsrepository/Azure.Iot.ModelsRepository/src/ModelMetadata.cs

```diff
@@ -6,6 +6,9 @@
 
 namespace Azure.Iot.ModelsRepository
 {
+    /// <summary>
+    /// <c>ModelMetadata</c> is designated to store KPI's from model parsing.
```

Comment:
Plural of KPI is KPIs. Possessive of KPI is KPI's. I think in this case you intend plural, yes?

Reply:
yea, will fix